packages/core/js/MobileCore.js

```diff
@@ -136,22 +136,22 @@ module.exports = {
    * @param event Required parameter with {@link Event} instance to be dispatched. Should not be nil
    * @return true if the the event dispatching operation succeeded, otherwise the promise will return an error
    */
-  dispatchEvent(event: AEPExtensionEvent): Promise<boolean> {
+  dispatchEvent(event: Event): Promise<boolean> {
```

Comment:
This API can be renamed to `dispatch(event: Event)`

Reply:
I think `dispatchEvent` is a better option for the loosely typed JavaScript. We also use this method name in Android SDK ->
```Java
public static boolean dispatchEvent(final Event event, final ExtensionErrorCallback<ExtensionError> errorCallback)
```